### PR TITLE
fix(stage-filmstrip) Fix active participant test

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Review by horymury — DISMISSED** [review]

File: src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java [hunk 1]
```diff
@@ -91,9 +91,18 @@ private void assertOneDominantSpeaker(WebDriver driver)
             driver.findElements(By.xpath(
                 "//span[contains(@class,'dominant-speaker')]"
             ));
+        List<WebElement> nonStage = new ArrayList<>();
+        for (WebElement el : dominantSpeakerIndicators)
```
Comment (hristoterezov) [inline src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java:95]: I'm wondering can we do the whole thing with XPath only?

Maybe we can have `id`s on the filmstrip containers and include in the XPath that there must be a parent with a specific `id` or maybe even use the current class of the stage filmstrip container in order to not include it in the XPath selection?

**Review by hristoterezov — CHANGES_REQUESTED** [review]
Left a comment and suggestion.

File: src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java (hunk 1 above)
Comment (robertpin) [reply src/test/java/org/jitsi/meet/test/ActiveSpeakerTest.java:95]: Yes, I think using xpath is a good idea. I'll update the code

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Comment by robertpin** [comment]
Jenkins test this please

**Review by hristoterezov — APPROVED** [review]

**Pull request merged**